index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> dog adoption</title>
+ <script src="https://kit.fontawesome.com/2315930367.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+
+  <link rel="stylesheet" type= "text/css" href= "./styles/styles.css">
+  <script src="./script.js"></script>
+</head>
+<body>
+  <header>
+
+    <div class= "checkout">
+      <i class="fas fa-shopping-cart"></i>
+      <button type="button" onclick="alert('adoptCart')">Checkout</button>
+  </div>
+<div class= "logo">
+    <img src="./images/adoptadog-logo.svg" alt="logo">
+    <h1>Seattle’s premiere dog adoption agency</h1>
+</div>
+  </header>
+  <nav>
+      <ul>
+        <li><a href="./index.html">Home/</a></li>
+        <li><a href="./all-dogs.html">Dogs/</a></li>
+        <li><a href="./blog.html">Blog/</a></li>
+      </ul>
+  </nav>
+
+
+<aside>
```

Comment:
when I said sidebar above I meant `<aside>` brain fart. Good use of this element